### ISSUE TITLE
Documented memcpy/memset/memmove methods

### ISSUE THREAD

**Issue by malte-v** [crystal-lang/crystal#7700]
Using `memcpy/memset/memmove` is quite common when dealing with C bindings. I know about the `Intrinsics` module, but as the name suggests it's undocumented and not really supposed to be used directly. I was thinking about making these functions accessible in `Pointer` and adding documentation for them there.

**Comment by oprypin** [comment]
The functionality is all present, e.g.

https://crystal-lang.org/api/0.28.0/Pointer.html#move_to%28target%3APointer%2Ccount%3AInt%29-instance-method

???

**Comment by malte-v** [comment]
My bad, I just searched the docs for `memcpy`.

**Comment by oprypin** [comment]
tbh, you could make a case for including the word "memmove" somewhere in documentation for just for the sake of searchability

**Comment by straight-shoota** [comment]
Yes, I think people should be able to find this feature when looking for memcpy/memset/memmove
